packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -75,4 +75,12 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Optional class name to apply to tooltip host.
    */
   hostClassName?: string;
+
+  /**
+   * Manual way to display tooltip when the element is not directly focusable but an ancestor is,
+   * so it can pass down that state via this property.
+   * It honors overflow rules and keyboard and mouse interaction, so this just adds another interaction way
+   * delegated on another element.
+   */
+  isFocused?: boolean;
```

Comment:
Took me a little while to understand what `isFocused` meant. How about isVisible or isForcedVisible.

Secondly, the tooltipHost was meant to remove the need to manually turn the tooltip on or off. Would it be better to simply use the Tooltip component, and control the hide/show yourself?

Reply:
I'm planning to use all what current TooltipHost offers: a tooltip displayed based on mouse and keyboard, plus an overflow mode to display it only when content exceeds. I need this all, but also a way to manually display the tooltip, which in React means converting this uncontrolled component into a controlled one via the `isFocused` property, which is not explicit in current implementation and that's why we're having a hard time naming it (it's more than keyboard focus because it would mean mouse over too).
So in order to create my own custom version, I need to duplicate the mouse/keyboard events plus the overflow logic, when all I need is a controlled version of TooltipHost, this is why I feel it's a natural move for this component.
**Note:** I admit my proposal is not a pure controlled version because local mouse/keyboard events are still considered, this was a convenience that also seems natural to me: no matter how the tooltip was displayed, when user interacts with it the tooltip should be hidden (which is TooltipHost built-in behavior).
That said, if you feel strong against this, I will understand if you just close this PR.

Reply:
More details after writing my custom component: I've had to duplicate all the logic from `TooltipHost` and then add the `isFocused` property. I cannot reuse `TooltipHost` because it doesn't allow altering its behavior from outside, it's strongly tight to local events (mouse and keyboard). So my custom version is exactly a copy of my proposal on this PR.

IMHO current design hurts reusability. Ideally it should be the other way around: a TooltipHost with a `isFocused` property, and then a wrapper that connects it to keyboard and mouse events.

Reply:
What if we rename it to `initialDisplayState` or something along those lines?

Reply:
It should not be `initial` because it's not intended as an initial value, but as a fully-managed prop.
If we want to narrow it down to my case, it could be `isAncestorFocused` which can stand for a more defined meaning. Or `isRelativeFocused`. Or `isHookedElementFocused`.

Reply:
but it's not fully managed? don't you override when I mouse over and out?

Reply:
Yep, that's a convenience because I believe the user should always be able to dismiss a tooltip by passing the mouse over it, but I would understand if you guys tell me to change it to be "truthfully" fully-managed.
Besides that, the property can be set many times during the life of the component and this have always to react to it, so it's not an `initial` value.

Reply:
I'd be okay with this approach if we made the tooltip host either controlled, or uncontrolled. I.E. if you are managing the state of the tooltip in the host, then you manage the state. Obviously you can switch between managed/unmanaged, but if the tooltip is managed, I'd want that logic to override the internal logic. 

I could see making `_onTooltipMouseEnter` a prop so that you could pass in your own function there, that way you can maintain state in your application rather than the tooltip.